legal-api/src/legal_api/services/filings/validations/incorporation_application.py

```diff
@@ -226,6 +234,59 @@ def validate_incorporation_effective_date(incorporation_json) -> Error:
 
     return None
 
+def validate_cooperative_documents(incorporation_json) -> Error:
+    """Return an error or warning message based on the cooperative documents validation rules.
+
+    Rules:
+        - The documents are provided.
+        - Document IDs are unique.
+    """
+    # Setup
+    msg = []
+
+    rules_file_key = incorporation_json['filing']['incorporationApplication']['cooperative']['rulesFileKey']
+    rules_file_name = incorporation_json['filing']['incorporationApplication']['cooperative']['rulesFileName']
+    memorandum_file_key = incorporation_json['filing']['incorporationApplication']['cooperative']['memorandumFileKey']
+    memorandum_file_name = incorporation_json['filing']['incorporationApplication']['cooperative']['memorandumFileName']
+
+    # Validate key values exist
+    if not rules_file_key:
+        msg.append({'error': babel('A valid rules key is required.')})
+
+    if not rules_file_name:
+        msg.append({'error': babel('A valid rules file name is required.')})
+
+    if not memorandum_file_key:
+        msg.append({'error': babel('A valid memorandum key is required.')})
+
+    if not memorandum_file_name:
+        msg.append({'error': babel('A valid memorandum file name is required.')})
+
+    if msg:
+        return msg
+
+    # Validate the files from Minio
+    try:
+        MinioService.get_file(rules_file_key)
```

Comment:
Do we need this get?

Reply:
We need to get the file to validate that the document page size is letter size.  You are going to add that code right Cam?

Reply:
Yea, I’ve genuinely struggled with this though. I haven’t figured out how to do this server side as of yet. Could def use some help here!

Reply:
yeah we can talk about this